src/base/net/downloadmanager.cpp

```diff
@@ -182,15 +182,14 @@ Net::DownloadHandler *Net::DownloadManager::download(const DownloadRequest &down
     return downloadHandler;
 }
 
-void Net::DownloadManager::registerSequentialService(const Net::ServiceID &serviceID, std::chrono::seconds delay)
+void Net::DownloadManager::registerSequentialService(const Net::ServiceID &serviceID, const std::chrono::seconds delay)
 {
-    setSequentialServiceDelay(serviceID ,delay);
-    m_sequentialServices.insert(serviceID);
+    m_sequentialServices.insert(serviceID, delay);
 }
 
-void Net::DownloadManager::setSequentialServiceDelay(const ServiceID &serviceID, std::chrono::seconds delay)
+void Net::DownloadManager::setSequentialServiceDelay(const ServiceID &serviceID, const std::chrono::seconds delay)
```

Comment:
It does exactly the same as `registerSequentialService()` does.

Reply:
Fixed with c62f38abca02a0396b6371d2d215d0bc01c53951
I removed setSequentialServiceDelay and replace refrences with registerSequentialService.
Didn't notice QHash insert updates value if key already exists